src/eko/io/runcards.py

```diff
@@ -376,3 +415,24 @@ def update(theory: Union[RawCard, TheoryCard], operator: Union[RawCard, Operator
 
     cards = Legacy(theory, operator)
     return cards.new_theory, cards.new_operator
+
+
+def flavored_mu2grid(mu2grid: list, masses: list, matching_ratios: list):
+    r"""Upgrade :math:`\mu^2` grid to contain also target number flavors.
+
+    It determines the number of flavors for the PDF set at the target scale,
+    inferring it according to the specified scales.
+
+    This method should not be used to write new runcards, but rather to have a
+    consistent default for comparison with other softwares and existing PDF
+    sets.
+    There is no one-to-one relation between number of running flavors and final
+    scales, unless matchings are all applied. But this is a custom choice,
+    since it is possible to have PDFs in different |FNS| at the same scales.
+
+    .. todo::
+
+        To be actually implemented
+
+    """
+    return [(np.sqrt(mu2), 0) for mu2 in mu2grid]
```

Comment:
```suggestion
    return [(np.sqrt(mu2), nan) for mu2 in mu2grid]
```
as for other quantities (e.g. `nfref`) let's use `nan` to autodetect

Reply:
It is just a placeholder, there is written right above `To be actually implemented`, and this is the first item missing in the OP

Reply:
Implemented in 7673b78